frontend/src/hocs/withScreencast/components/VideoRecordedDialog/VideoRecordedDialog.tsx

```diff
@@ -15,95 +15,85 @@
  * limitations under the License.
  */
 
-import React, { PureComponent, ReactNode } from "react"
-import { Button } from "reactstrap"
+import React, { FunctionComponent } from "react"
+import Button, { Kind } from "components/shared/Button"
 import Modal, { ModalHeader, ModalBody } from "components/shared/Modal"
 
 import "./style.scss"
 
 export interface Props {
   /** Callback to close the dialog */
   onClose: () => void
-
   videoBlob: Blob
 
   fileName: string
 }
-
-class VideoRecordedDialog extends PureComponent<Props> {
-  handleDownloadButton = (): void => {
-    const { onClose } = this.props
+const VideoRecordedDialog: FunctionComponent<Props> = ({
+  onClose,
+  videoBlob,
+  fileName,
+}) => {
+  const videoSource = URL.createObjectURL(videoBlob)
+  const handleDownloadClick: () => void = () => {
+    // Downloads are only done on links, so create a hidden one and click it
```

Comment:
Curious, why didn't we need this "create an invisible link" logic before?

Reply:
Basically the `download` attribute can only be applied to links (ie `<a>` tags) ReactStrap had this ability to create a link instead of a button if the href is specified https://github.com/reactstrap/reactstrap/blob/master/src/Button.js#L78

Reply:
Ah, that makes sense. Thanks!